src/main/webapp/search.js

```diff
@@ -136,6 +143,59 @@ function buildProjectCard(title, tags, author, image){
 
 }
 
+function getSingleProject(projectId){
```

Comment:
Well done and very well documented with the comments!

Reply:
Thanks!